frontend/src/entry/model/Releases.tsx

```diff
@@ -50,6 +50,10 @@ export default function Releases({ model, currentUserRoles, readOnly = false }:
     return <MessageAlert message={isReleasesError.info.message} severity='error' />
   }
 
+  if (isReleasesLoading) {
```

Comment:
I don't see why we need to hide the draft button when the releases are loading. Can we revert this back to where it was?